sites/public/styles/overrides.scss

```diff
@@ -155,6 +155,10 @@
     }
   }
 
+  .seeds-dialog {
```

Comment:
Fixes a bug with seeds dialogs that causes mobile navigation bars to overlap page content. We need to bring this over to seeds, but I couldn't test it fully without fixing it here for now.